package.json

```diff
@@ -2,16 +2,22 @@
   "name": "lke-plugin-csv-importer",
   "version": "0.0.1",
   "description": "A plugin to quickly import CSV data into LKE",
-  "main": "index.js",
+  "main": "public/index.js",
   "scripts": {
     "tsc": "tsc -b",
     "clean": "rm -rf csv-importer.lke",
-    "build": "npm-run-all clean build:public build:backend copyManifest",
+    "build:project": "npm-run-all clean build:public build:backend copyManifest",
     "build:public": "tsc -b src/public && npm run copy-assets && cd src/public/dist && browserify *.js --debug -o ../../../csv-importer.lke/public/index.js",
     "copy-assets": "cp -R src/public/assets csv-importer.lke/public && cp src/public/index.html csv-importer.lke/public",
     "build:backend": "tsc -b src/backend && npm run installProduction",
     "installProduction": "cp package.json csv-importer.lke && cd csv-importer.lke",
     "copyManifest": "cp -R src/manifest.json csv-importer.lke/",
+    "lint:ci-deactivated": "eslint -f checkstyle -o reports/checkstyle.xml  .",
+    "build": "npm run build:project && npm pack && gunzip -qf -c linkurious-lke-csv-importer-$(cat .version).tgz > lke-plugin-csv-importer.lke",
```

Comment:
linkurious-lke-csv-importer- is not the name of the package, it should be: lke-plugin-csv-importer-

Reply:
Please don't run clean on build.